### PR TITLE
AS-915: Remove UKBB from the Data Library Page

### PR DESCRIPTION
**Pull request by ansingh7115**
<!--
Hello, friend!
Remember to mention what sort of testing/verification you performed in the course of building this PR, i.e. checking functionality in the browser locally.
Thanks!
--!>

### CODE REVIEW

File: src/libs/logos.js [hunk 1]
```diff
@@ -12,10 +12,8 @@ import fcLogoWhite from 'src/images/brands/firecloud/FireCloud-Logo-White.svg'
 import terraLogo from 'src/images/brands/terra/logo.svg'
 import terraLogoWhite from 'src/images/brands/terra/logo-grey.svg'
 import terraLogoShadow from 'src/images/brands/terra/logo-wShadow.svg'
-import ukbLogo from 'src/images/brands/ukbiobank/uk-biobank-logo-color.svg'
-import ukbLogoWhite from 'src/images/brands/ukbiobank/uk-biobank-logo-white.svg'
```
Comment (davidangb) [inline src/libs/logos.js:16]: nit: also delete the `src/images/brands/ukbiobank/` directory

File: src/pages/library/Datasets.js [hunk 2]
```diff
@@ -18,7 +18,6 @@ import nemoLogo from 'src/images/library/datasets/nemo-logo.svg'
 import targetLogo from 'src/images/library/datasets/target_logo.jpeg'
 import tcgaLogo from 'src/images/library/datasets/TCGALogo.jpg'
 import topMedLogo from 'src/images/library/datasets/TopMed@2x.png'
-import ukbLogo from 'src/images/library/datasets/UKB@2x.jpg'
```
Comment (davidangb) [inline src/pages/library/Datasets.js:21]: nit: also delete `src/images/library/datasets/UKB@2x.jpg`

**Review by davidangb — COMMENTED** [review]
two nits, else :+1:

**Review by kyuksel — APPROVED** [review]
LGTM  👍🏼

File: src/libs/logos.js (hunk 1 above)
Comment (ansingh7115) [reply src/libs/logos.js:16]: good catch!

**Review by davidangb — APPROVED** [review]

**Review by petesantos — APPROVED** [review]

**Pull request merged**